programs/sbf/rust/syscall-get-epoch-stake/src/lib.rs

```diff
@@ -1,17 +1,18 @@
 //! Example Rust-based SBF program that tests the `sol_get_epoch_stake`
 //! syscall.
 
-extern crate solana_program;
-use solana_program::{
-    account_info::AccountInfo,
-    entrypoint::ProgramResult,
-    epoch_stake::{get_epoch_stake_for_vote_account, get_epoch_total_stake},
-    msg,
-    program::set_return_data,
-    pubkey::Pubkey,
+use {
+    solana_account_info::AccountInfo,
+    solana_msg::msg,
+    solana_program::{
+        epoch_stake::{get_epoch_stake_for_vote_account, get_epoch_total_stake},
```

Comment:
Eep, these need to get migrated somewhere: https://github.com/anza-xyz/solana-sdk/issues/178